mava/logger.py

```diff
@@ -74,6 +75,8 @@ def log(
                 float(np.mean(episodes_length)),
                 t_env,
             )
+            logger.log_stat(prefix.lower() + "steps_per_second", steps_per_second, t_env)
+
             if trainer_metric:
                 logger.log_stat("total_loss", float(np.mean(total_loss)), t_env)
                 logger.log_stat("value_loss", float(np.mean(value_loss)), t_env)
```

Comment:
What do you think about adding a prefix to these info? I'm not sure if it would be beneficial.

Reply:
What would the prefix be?

Reply:
maybe "trainer/" even though these only measured for the trainer :sweat_smile: ?